packages/icons/src/ExampleIcon.test.tsx

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import { render, screen } from '@testing-library/react';
+import { ExampleIcon } from './ExampleIcon';
```

Comment:
Why not just use one of the existing icons?

Reply:
Because the exclude from the icons folders by jest. How to easily choose an icon without change the whole folder structure? 
You have a better idea? I am not sure about this and then chose this way.

Reply:
Wer only exclude for coverage

Reply:
We are also exclude for jest in config/jest/index.js
What should I do? 
- Make exclude a little bit more complicated 
- or a seperate Example Icon like in this PR? 
@sebald @viktoria-schwarz

Reply:
Can you give me the line where it is ignored? 🤔

Reply:
Sorry, my fault.